CI/travis/inside_bionic_docker.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh -xe
+
+TRAVIS_CI="$1"
+
+apt-get -qq update
+apt-get install -y cmake doxygen libaio-dev libavahi-client-dev libavahi-common-dev libusb-1.0-0-dev libxml2-dev git
+apt-get install -y tar bzip2 gzip flex bison
```

Comment:
you could include the `before_install_linux` script here ;
for Ubuntu this is do-able, since it's common code with the main code that runs in Travis-CI